node_modules/oae-core/signin/css/signin.css

```diff
@@ -0,0 +1,61 @@
+/*!
+ * Copyright 2015 Apereo Foundation (AF) Licensed under the
+ * Educational Community License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may
+ * obtain a copy of the License at
+ *
+ *     http://opensource.org/licenses/ECL-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an "AS IS"
+ * BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+#signin-modal {
+    text-align: center;
+}
+
+#signin-modal > .modal-dialog {
+    display: inline-block;
+    min-width: 300px;
+    text-align: left;
+    width: auto;
+}
+
+#signin-modal > .modal-dialog .modal-header {
+    padding-right: 12px;
```

Comment:
I don't remember doing this, but probably the Close icon must have looked out of place when it came closer to the header content -- this is one of our skinnier modals. Looking at the profile picture modal it looks out of place too. It's either too high up or the assymetric padding on top and right looks funny.

![image](https://cloud.githubusercontent.com/assets/102265/9865497/46b9c14a-5b25-11e5-9c88-a8ca851f95e2.png)

![image](https://cloud.githubusercontent.com/assets/102265/9865502/5c489c52-5b25-11e5-85c2-b6d2cda0568b.png)


We should do it to all or none though, I'll just take it out of this one.